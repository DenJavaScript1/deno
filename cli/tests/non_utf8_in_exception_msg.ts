throw new Error(new TextDecoder().decode(new Uint8Array([255])));
