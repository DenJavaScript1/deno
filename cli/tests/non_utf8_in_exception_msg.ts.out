[WILDCARD]error: Uncaught Error: ï¿½
throw new Error(new TextDecoder().decode(new Uint8Array([255])));
      ^
    at [WILDCARD]non_utf8_in_exception_msg.ts:1:7
